hack/release/prepare-assets.sh

```diff
@@ -107,6 +107,8 @@ export IMG_TAG=$VERSION
 export IMG_NAME=projects.registry.vmware.com/antrea/antrea-ubuntu
 ./hack/generate-standard-manifests.sh --mode release --out "$OUTPUT_DIR"
 
+sed "s|antrea\/antrea-migrator:latest|$(echo $IMG_NAME | sed "s/ubuntu/migrator/"):$VERSION|g" ./build/yamls/antrea-migrator.yml > "$OUTPUT_DIR"/antrea-migrator.yml
```

Comment:
Could you use "projects.registry.vmware.com/antrea/antrea-migrator:$VERSION" directly to replace `antrea-migrator:latest`? There is an on-going change #5794 to split agent and controller images. Maybe better to have your own image string in case there is conflict.

Reply:
Removed this line since we do not need to add it to release asserts.